target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf289f.dts

```diff
@@ -0,0 +1,435 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+        model = "ZTE MF289F";
+        compatible = "zte,mf289f";
+
+        aliases {
+                led-boot = &led_status;
+                led-failsafe = &led_status;
+                led-running = &led_status;
+                led-upgrade = &led_status;
+        };
+
+        chosen {
+                /*
+                 * bootargs forced by u-boot bootipq command:
+                 * 'ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait'
+                 */
+                bootargs-append = " root=/dev/ubiblock0_1";
+        };
+
+        /* This node is used to restart modem module to avoid anomalous behaviours on initial communication  */
+        gpio-restart {
+                compatible = "gpio-restart";
+                gpios = <&tlmm 8 GPIO_ACTIVE_HIGH>;
+        };
+
+        leds {
+                compatible = "gpio-leds";
+
+                led_status: led-status {
+                        function = LED_FUNCTION_STATUS;
+                        color = <LED_COLOR_ID_BLUE>;
+                        gpios = <&tlmm 10 GPIO_ACTIVE_LOW>;
+                        default-state = "keep";

```

Comment:
Maybe `default-state = "on";` "keep" does nothing here. Please also add `label = "blue:status";`.

Reply:
fixed, please refer to new PR: https://github.com/openwrt/openwrt/pull/10708